### PR TITLE
rsweb-8229-deloitte-rug

### PR DESCRIPTION
**Pull request by amberRrucker**
Viewable at the bottom of the homepage:
http://localhost:9000/derek/incubation/homepage

Looking to replace the rug with something a little more useful. Images are in the ticket.

### CODE REVIEW

File: styleguide/_themes/global/scss/vendor_prefixes.scss [hunk 1]
```diff
@@ -41,12 +41,6 @@
   border-radius: $radius;
 }
 
-@mixin rotate($degrees) {
-  -webkit-transform: rotate($degrees);
-  -ms-transform: rotate($degrees);
-  transform: rotate($degrees);
-}
-
```
Comment (amberRrucker) [inline styleguide/_themes/global/scss/vendor_prefixes.scss]: this is a baked in bootstrap mixin so i removed it from our code.

**Comment by WidgetsBurritos** [comment]
image

Should there be more left padding when we're ready at the 992px breakpoint?

**Comment by WidgetsBurritos** [comment]
image

Same for 1280

File: styleguide/derek/_partials/_rug.ejs [hunk 2]
```diff
@@ -1,45 +1,48 @@
...
+            <i class="fa fa-envelope-o fa-2x rug-icon"></i>
           </div>
-          <div class="col-md-8 col-lg-9 rug-link rug-link-hover rug-email-text">
+          <div class="rug-link rug-link-hover">
             Email Us
           </div>
         </div>
       </a>
     </div>
+  <div class="rug-headerWrapper rug-headerWrapperBorder">
```
Comment (WidgetsBurritos) [inline styleguide/derek/_partials/_rug.ejs]: fix the indention here

**Comment by amberRrucker** [comment]
@WidgetsBurritos the padding on the left should be the same as the footer links. Theres not actually any right hand padding. I'm not sure on the mini panel stuff. I just kept the container with the padding and background, because the color should be standard across all quotes and it was already on all of them so I didn't want there to be more things to change in the CMS.

File: styleguide/derek/_partials/_rug.ejs [hunk 3]
```diff
@@ -1,45 +1,48 @@
...
+      <h4 class="rug-head">Not Sure Where to Start?</h4>
+      <p class="rug-subhead">Choose your preferred contact method</p>
     </div>
-    <div class="col-md-3">
+    <div class="rug-contactOption">
       <div class="row">
-        <div class="col-md-3 col-lg-2">
-          <i class="fa fa-mobile fa-4x rug-icon"></i>
+        <div class="rug-iconWrapper">
+          <i class="fa fa-phone fa-2x rug-icon  rug-iconPhone"></i>
```
Comment (WidgetsBurritos) [inline styleguide/derek/_partials/_rug.ejs]: Can you remove one of the extra spaces here?

File: styleguide/derek/_partials/_rug.ejs [hunk 4]
```diff
@@ -1,45 +1,48 @@
 <!--[rug]-->
-<div class="hidden-xs hidden-sm rug">
-  <div class="container rug-container">
-    <div class="col-md-3">
-      <h5 class="rug-head">We've got your back</h5>
-      <h4 class="rug-subhead">24<small>x</small>7<small>x</small>365</h4>
+<div class="rug">
+  <div class="rug-container">
```
Comment (WidgetsBurritos) [inline styleguide/derek/_partials/_rug.ejs]: To keep consistent with other patterns, this should probably be `rug-wrapper`

**Comment by WidgetsBurritos** [comment]
@amberRrucker okay that is fine.  Few minor tweaks then and I think you should be good.

**Comment by amberRrucker** [comment]
@WidgetsBurritos I addressed your feedback, aside from the padding because thats intentional.

**Comment by amberRrucker** [comment]
@runrog do you mind taking a look

**Pull request merged**